.github/workflows/R_CMD_CHECK.yml

```
@@ -79,6 +79,13 @@ jobs:
           extra-packages: any::rcmdcheck
           needs: check
 
+      - name: Apply duckdb-r patches
+        shell: bash
+        run: |
+          for filename in $GITHUB_WORKSPACE/$DUCKDB_SRC/.github/patches/duckdb-r/*.patch; do
```

```suggestion
          for filename in ${{ env.DUCKDB_SRC }}/.github/patches/duckdb-r/*.patch; do
```
(not that I am completely sure, but this seems to make more sense)

---

`$GITHUB_WORKSPACE` is just the absolute path. we also use ` $GITHUB_WORKSPACE/$DUCKDB_SRC/` [here](https://github.com/duckdb/duckdb/blob/8580bf103131b085c4f536a68e09741ec4edd7f1/.github/workflows/R_CMD_CHECK.yml#L102)

---

nvm, also realized that this was executing in the wrong directory

---

I got a bit lost with the directories, unsure.

Actually one thing you need to add, to handle the case of no files being there, is:
```
shopt -s nullglob
```

See for example this part of the Nightly workflow: https://github.com/duckdb/duckdb/blob/main/.github/workflows/NightlyTests.yml#L630